packages/components/src/alignment-matrix-control/index.tsx

```diff
@@ -61,31 +50,31 @@ export function AlignmentMatrixControl( {
 	label = __( 'Alignment Matrix Control' ),
 	defaultValue = 'center center',
 	value,
-	onChange = noop,
+	onChange,
 	width = 92,
 	...props
 }: WordPressComponentProps< AlignmentMatrixControlProps, 'div', false > ) {
-	const [ immutableDefaultValue ] = useState( value ?? defaultValue );
-	const baseId = useBaseId( id );
-	const initialCurrentId = getItemId( baseId, immutableDefaultValue );
+	const baseId = useInstanceId(
+		AlignmentMatrixControl,
+		'alignment-matrix-control',
+		id
+	);
 
-	const composite = useCompositeState( {
-		baseId,
-		currentId: initialCurrentId,
+	const compositeStore = useCompositeStore( {
+		defaultActiveId: getItemId( baseId, defaultValue ),
+		activeId: value && getItemId( baseId, value ),
```

Comment:
Is the `value &&` part necessary? Afterall, it look like the `activeId` prop in `useCompositeStore` also accepts `undefined`

Reply:
At one stage I was definitely having problems without the additional `value &&` here, but I can't reproduce that now. Past me should have added a comment, clearly!